a1q3.cpp

```diff
@@ -1,24 +1,26 @@
-/* name: vinayak sharma
-smvdu, jammu ECE
-code description: user enters 3 numbers. the 3 numbers get compared, if a>b>c then result is diplayed as 1 or else 0. */
+/* name: Ritwik Mishra

```

Comment:
Kindly remove this

Reply:
Removed it in another pull request.
(Couldn't find how to do it from here only)